src/test/resources/Login.feature

```diff
@@ -1,12 +1,23 @@
-Feature: Login
-
-  An a user
-  I want to sign in
-  So I can use the app
-
-  @automated
-  Scenario: User can sign in with valid value
-    Given user is in Katalon login page1
-    When user sets username as "John Doe" and password as "ThisIsNotAPassword"
-    And user clicks on login button
-    Then user sees appointment page
+Feature: Login
+
+  As an user
+  I want to sign in
+  So I can use the app
+
+  @automated
+  Scenario: User can sign in with valid value
+    Given user is in Katalon login page
+    When user sets username as "John Doe" and password as "ThisIsNotAPassword"
+    And user clicks on login button
+    Then user sees appointment page
+
+  Scenario Outline: User can sign in with invalid value
```

Comment:
` User can sign in with invalid value `=>` User cannot sign in with invalid value`